locations/spiders/att.py

```diff
@@ -1,6 +1,8 @@
-import scrapy
 import json
 import re
+
+import scrapy
```

Comment:
Can you put this line back at the top of the file?

Reply:
Will do!